### PR TITLE
Homework 14 and Exam

### PR DESCRIPTION
**Pull request by S-Arashu**

### CODE REVIEW

**Review by kino1ol — COMMENTED** [review]
All where use link or onclick add cursor pointer.

File: AnastasiyaTomasheva/JS/HW_14_and_exam/index.js [hunk 1]
```diff
@@ -0,0 +1,736 @@
...
+            `
+        let cardBlock = document.querySelector('.container_main');
+            cardBlock.prepend(card);
+
+        let load = document.cookie.split(';');
+            
+            for (let j = 0; j < load.length; j++) {
+                let cookie = load[j].split('=');
+
+                    if(listOfProducts[i].id == cookie[0]){
```
Comment (kino1ol) [inline AnastasiyaTomasheva/JS/HW_14_and_exam/index.js:130]: if id is number and cookie[0] is number use strict equal

File: AnastasiyaTomasheva/JS/HW_14_and_exam/index.js [hunk 2]
```diff
@@ -0,0 +1,736 @@
...
+setInterval(() => {
+
+    if(currentImageCount != imagesAreaImages.length){
+        currentImageCount++;
+        sliderController();
+    }else{ 
+        currentImageCount = 1;
+        sliderController();
+    };
+}, 3000);
```
Comment (kino1ol) [inline AnastasiyaTomasheva/JS/HW_14_and_exam/index.js:736]: very big file is 700 row. you should split into modules according to a functional parts

**Review by kino1ol — APPROVED** [review]
your ball is 9

**Comment by kino1ol** [comment]
@S-Arashu fix conflicts please for merge in master branch
This branch has conflicts that must be resolved
 to resolve conflicts before continuing.
Conflicting files
AnastasiyaTomasheva/.DS_Store
AnastasiyaTomasheva/JS/.DS_Store

**Comment by S-Arashu** [comment]
@kino1ol It's done)

**Pull request merged**